### DISCUSSION TITLE
Is TTFB included in LCP?

### DISCUSSION THREAD

**Discussion by ghisleouf** [GoogleChrome/lighthouse#15763] [Q&A] [answered]
**Summary**
Hi there, 
Happy New Year ! (it's not yet too late !) 🎉

I'm having a question about LCP calculation.
Regarding the following diagram I got from some WebPerf experts, I'm quite perplexed about how the LCP is calculated.

<img width="1065" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/1439092/06439314-165b-45a2-a5da-ddfe27ca5074">

As far as I can see in this diagram, TTFB is not taken into account in LCP calculation ... but, after performing several tests on Webpagetest for example, what I can see is if my TTFB is much more shorter (the test I'm running are performed on a website with very light caching), LCP is reduced drastically:
- I started with a TTFB of 4s (no cache warmup) -> LCP is 7s
- With a low TTFB (300ms) -> LCP is < 3s

So, my question is "Is TTFB taken into account is LCP duration and calculation?"

Thanks a lot for your help. 🙏

Best regards.

**Comment by adamraine** [comment, marked as answer]
TTFB is included in the LCP time according to [Chrome's LCP definition](https://web.dev/articles/lcp#what_is_lcp). I believe the above diagram is misleading, where did you find it?

**Comment by ghisleouf** [comment]
Hi @adamraine ,

Thank you for your confirmation. It makes sense.
To be honest, I get this diagram from a Google WebPerf expert :)

I'm gonna update it.

Best,
